src/fairseq2/models/wav2vec2/frontend.py

```diff
@@ -129,15 +129,15 @@ def forward(
                 "`Wav2Vec2Frontend` does not support incremental decoding."
             )
 
-        seqs, padding_mask = self.extract_features(seqs, padding_mask)
+        seqs, padding_mask, _ = self.extract_features(seqs, padding_mask)
 
         seqs, padding_mask, _ = self.process_features(seqs, padding_mask)
 
         return seqs, padding_mask
 
     def extract_features(
         self, seqs: Tensor, padding_mask: Optional[PaddingMask]
-    ) -> Tuple[Tensor, Optional[PaddingMask]]:
+    ) -> Tuple[Tensor, Optional[PaddingMask], Tensor]:
```

Comment:
We should coordinate with Arrival since this would be a breaking change. @shagunsodhani